etc/disable-common.inc

```diff
@@ -268,6 +268,9 @@ read-only ${HOME}/.gem
 read-only ${HOME}/.luarocks
 read-only ${HOME}/.npm-packages
 read-only ${HOME}/bin
+read-only ${HOME}/.bin
+read-only ${HOME}/.local/bin
+read-only ${HOME}/.cargo/bin
```

Comment:
why not just make all of cargo readonly?

Reply:
Idea:
```
$ ls .cargo
bin/  config  env  registry/
```
config and registry are only needed by cargo it self, so we can blacklist it.
env is maybe used by some shell config files (e.g. .profile, .bashrc,...). only read-only.

Reply:
I don't rust, so I'll trust your judgement.

Reply:
OK, after a short look, if we blacklist `registry` we muss update atom, gnome-builder, bracktes, maybe vim and emacs, in the future geany. If this is ok i will update these profiles tomorrow. It's getting late for me ;)

Reply:
OK, I've made some changes.